Sources/App/Commands/TriggerBuilds.swift

```diff
@@ -484,13 +484,13 @@ func trimBuilds(on database: Database) async throws -> Int {
         FROM builds b
         USING versions v
         WHERE b.version_id = v.id
-        AND b.created_at < NOW() - INTERVAL '\(raw: String(Constants.trimBuildsGracePeriod.inHours)) hours'
+        AND b.created_at < NOW() - INTERVAL '\(unsafeRaw: String(Constants.trimBuildsGracePeriod.inHours)) hours'
```

Comment:
I don't think this `unsafeRaw` can be eliminated. I tried

```swift
        AND b.created_at < NOW() - INTERVAL \(bind: "\(Constants.trimBuildsGracePeriod.inHours) hours")
```

and

```swift
        AND b.created_at < NOW() - INTERVAL '\(bind: "\(Constants.trimBuildsGracePeriod.inHours) hours'
```

and variants like it but that breaks the query, not because it doesn't parse/run but because it doesn't work. We discovered this in https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/909 and have a specific test for this in `test_trimBuilds_bindParam`.

cc @gwynne